apps/tools/mkkconfig.sh

```diff
@@ -62,12 +62,18 @@ USAGE () {
   echo "   Identifies the specific configuratin for the selected <board-name>."
   echo "   This must correspond to a sub-directory under the board directory at"
   echo "   under build/configs/<board-name>/."
+  echo " -mindepth <mindepth>:"
+  echo "   Set the minimum depth to find configurations in sub-directory"
+  echo " -maxdepth <maxdepth>:"
+  echo "   Set the maximum depth to find configurations in sub-directory"
   echo " <-h>:"
   echo "   Prints this message and exits."
 }
 
 KCONFIG=Kconfig
 unset MENU
+DEPTH_MIN=2
+DEPTH_MAX=2
```

Comment:
How about initializing the max as 3?

Reply:
Is there any special reason for initial value?
This script is used for making configuration list of apps by apps/Makefile also.
So I think initialization of max value as 3 is not meaningful.